examples/02-registered-shopper-auth-token.ts

```diff
@@ -8,74 +8,52 @@
 /**
  * Get authorization token for a registered customer
  * Usage: ts-node examples/02-registered-shopper-auth-token.ts
- * Note: Replace configuration parameters and user credentials before running
  */
-import { ClientConfig, Customer } from "../dist";
-import {
-  getObjectFromResponse,
-  ResponseError,
-  ShopperToken,
-  stripBearer,
-} from "@commerce-apps/core";
-import ShopperCustomers = Customer.ShopperCustomers;
+import { ClientConfig, Customer } from "commerce-sdk";
 
-//client configuration parameters
+// client credentials
+const CLIENT_ID = "da422690-7800-41d1-8ee4-3ce983961078";
+const CLIENT_SECRET = "D*HHUrgO2%qADp2JTIUi";
+const ORG_ID = "f_ecom_zzte_053";
+const SHORT_CODE = "kv7kzm78";
+const SITE_ID = "RefArch";
+
+// client configuration parameters
 const clientConfig: ClientConfig = {
   parameters: {
-    clientId: "your-client-id",
-    organizationId: "your-org-id",
-    shortCode: "your-short-code",
-    siteId: "your-site-id",
+    clientId: CLIENT_ID,
+    organizationId: ORG_ID,
+    shortCode: SHORT_CODE,
+    siteId: SITE_ID,
   },
 };
 
-//customer credentials
-const username = "username";
-const password = "password";
-
 /**
  * Get token for the registered customer
  *
  * @returns authorization token
  */
-async function getRegisteredShopperToken(): Promise<
-  ShopperToken<ShopperCustomers.Customer>
-> {
-  const credentials = `${username}:${password}`;
-  const buff = Buffer.from(credentials);
-  const base64data = buff.toString("base64");
+async function getRegisteredShopperToken(): Promise<Customer.ShopperLogin.TokenResponse> {
+  const credentials = `${CLIENT_ID}:${CLIENT_SECRET}`;
+  const base64data = Buffer.from(credentials).toString("base64");
   const headers = { Authorization: `Basic ${base64data}` };
+  const client = new Customer.ShopperLogin(clientConfig);
 
-  const client = new ShopperCustomers(clientConfig);
-
-  const response = await client.authorizeCustomer(
-    { headers: headers, body: { type: "credentials" } },
-    true
-  );
-  if (!response.ok) {
-    throw new ResponseError(response);
-  }
-  const customerInfo: ShopperCustomers.Customer = await getObjectFromResponse(
-    response
-  );
+  const response: Customer.ShopperLogin.TokenResponse =
+    await client.getAccessToken({
+      headers,
+      body: {
+        grant_type: "client_credentials",
+      },
+    });
```

Comment:
```suggestion
  const response = await client.getAccessToken({
    headers,
    body: {
      grant_type: "client_credentials",
    },
  });
```
TypeScript is smart. You don't need to tell it the type.